test-framework/src/main/java/org/apache/brooklyn/test/framework/TestHttpCall.java

```diff
@@ -32,9 +36,34 @@
     @SetFromFlag(nullable = false)
     ConfigKey<String> TARGET_URL = ConfigKeys.newStringConfigKey("url", "URL to test");
 
+    @SetFromFlag(nullable = false)
+    ConfigKey<HttpMethod> TARGET_METHOD = ConfigKeys.builder(HttpMethod.class)
+            .name("method")
+            .description("Method to request the URL: GET, POST, PUT, DELETE, etc")
+            .defaultValue(HttpMethod.GET)
+            .build();
+
+    @SetFromFlag
```

Comment:
But if I remove the `@SetFromFlag`, we won't be able to set this config key outside the `brooklyn.config` block, isn't it @aledsage ?

I also keep them as it was the "standard" across the test entities